package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "atb",
-  "version": "1.48.0",
+  "version": "1.49.0",
```

Comment:
@tormoseng I bumped the version for e2e tests as well. Should we add that to the checklist so it's done every time?

Reply:
Would be nice!